.travis.yml

```diff
@@ -31,4 +31,4 @@ script:
   - cd build
   - cmake -DBUILD_SHARED_LIBS=$CMAKE_SHARED -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DCMAKE_C_FLAGS="-Wall -Wextra -Werror" ..
   - cat config.h
-  - make && make test
+  - cmake --build . --config $BUILD_TYPE && ctest --output-on-failure -C $BUILD_TYPE
```

Comment:
That disables the autotool build in favor of the CMake build. Not a good idea.

Reply:
No. The autotool build is still done but in a completely different job: https://github.com/erikd/libsamplerate/pull/82/files#diff-354f30a63fb0907d4ad57269548329e3R16

Reply:
Ok, the way this travis file looks now, it seems like CMake is the default, when it is not. Autotools is the default.

Reply:
First: Why? Isn't CMake superior to autotools? It can generate buildsystems on any OS.

In this case it kinda has to be this way: All paths (windows, linux, optimized, non-optimized) go to the "default" `script` tag so the code there has to be the os-agnostic, generic code. Doing it differently in an attempt to make the autotools job more prominent would obfuscate the code (e.g. use yaml anchors)

Reply:
> Isn't CMake superior to autotools?

No, I don't think so and I know I am not alone.

I have used Autotools for 20 years. Its not perfect, but I understand it and know how to use it. I don't know CMake anywhere near as well, and from what I have seen of it, I do not think it is as good as Autotools.

Reply:
I personally like CMake more, especially if i look at what is required for autotools: https://github.com/erikd/libsamplerate/blob/master/Makefile.am, https://github.com/erikd/libsamplerate/blob/master/autogen.sh, https://github.com/erikd/libsamplerate/blob/master/configure.ac compared to CMake https://github.com/erikd/libsamplerate/blob/master/CMakeLists.txt.

But I don't want to start a philosophic discussing about meta-buildsystems. So only the advantages of CMake in this case:
- Generating build systems for windows (e.g. MSVC projects)
- Use of the library as a git submodule without the need to install it from a CMake project via `add_subdirectory`. This also greatly reduces the pain of using this on Windows/cross-platform projects with all the dependencies on compilers, options, ...

But here all this seems to be void. Why talk about a "default"? Both work are thoroughly tested and a user can choose which one he wants/needs (if he isn't on windows in which case there is no choice)